scripts/provisioning/s3setup/test_s3_setup.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+# This is a temporary script which can be used to run s3_setup for single
+# node again and again.
+
+die_with_error () {
+	echo $1
+	exit -1
+}
+
+./clean_open_ldap_by_s3.sh
+
+if [ ! -e "/opt/seagate/cortx/s3/conf/s3_confstore.json" ]; then
+	tee -a /opt/seagate/cortx/s3/conf/s3_confstore.json << END
```

Comment:
we should generate this file, not at install path, may be in tmp and delete it later.

Reply:
Addressed this comment.